app/backend/src/tests/test_account.py

```diff
@@ -422,6 +423,46 @@ def test_ChangeEmail_wrong_token(db, fast_passwords):
         assert user_updated.email == user.email
 
 
+def test_ChangeEmail_tokens_expire(db):
+    def two_hours_one_minute_in_future():
+        return now() + timedelta(hours=2, minutes=1)
+
+    new_email = f"{random_hex()}@couchers.org.invalid"
+    user, token = generate_user(hashed_password=None)
+
+    with account_session(token) as account:
+        account.ChangeEmail(
+            account_pb2.ChangeEmailReq(
+                new_email=new_email,
+            )
+        )
+
+    with session_scope() as session:
+        user = session.query(User).filter(User.id == user.id).one()
+        old_email_token = user.old_email_token
+        new_email_token = user.new_email_token
+
+    with patch("couchers.utils.now", two_hours_one_minute_in_future):
```

Comment:
To fix the import, you should be able to do something like `with patch("couchers.servicers.auth.now")`, and then you can import normally.